<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>471</width>
      <height>251</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Телефон</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Number">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>+7(999)-999-99-99</string>
    </property>
    <property name="text">
     <string>+7()---</string>
    </property>
    <property name="cursorPosition">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Имя</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
